scripts/convert_to_tarred_audio_dataset.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script converts an existing audio dataset with a manifest to
+# a tarred and sharded audio dataset that can be read by the
+# TarredAudioToTextDataLayer.
+
+import argparse
+import json
+import os
+import random
+import tarfile
```

Comment:
is tarfile in our requirements anywhere?

Reply:
Nope, added now! (To the ASR requirements--reasoning is that this script is specifically for converting ASR datasets.)

Reply:
...Actually, it looks like it's part of the standard library. Going to remove it.